docs/reference/commandline/build.md

```diff
@@ -457,3 +459,115 @@ space.
 storing two copies of the image, one for the build cache with all the cache
 layers in tact, and one for the squashed version.
 
+#### Pre-Requisites
+
+The example on this page is using experimental mode in Docker 1.13 that can be enabled by using an experimental flag or option to Docker daemon, users can start the Docker daemon with or without experimental features.
```

Comment:
This could use some rewriting. s/Docker 1.13 that can be enabled by using/Docker 1.13. Experimental mode can be enabled by using the `--experimental` flag when starting the Docker daemon or setting `experimental: true` in the `daemon.json` configuration file.

Or something like that. Basically, divide the sentence up a bit.